src/factories/workflow-specific/PIM_WorkflowFactory_v1.sol

```diff
@@ -0,0 +1,289 @@
+
+        // enable bonding curve to mint issuance token
+        issuanceToken.setMinter(fundingManager, true);
+
+        // transfer initial collateral supply from msg.sender to  bonding curve and mint issuance token to recipient
+        _manageInitialSupplies(
+            IBondingCurveBase_v1(fundingManager),
+            IERC20(PIMConfig.collateralToken),
+            issuanceToken,
+            PIMConfig.bcProperties.initialCollateralSupply,
+            PIMConfig.bcProperties.initialIssuanceSupply,
+            PIMConfig.recipient,
+            PIMConfig.withInitialLiquidity
+        );
+
+        // if applicable make first purchase
+        _manageInitialPurchase(
+            IBondingCurveBase_v1(fundingManager),
+            IERC20(PIMConfig.collateralToken),
+            PIMConfig.firstCollateralIn,
+            PIMConfig.recipient
+        );
+
+        // disable factory to mint issuance token
+        issuanceToken.setMinter(address(this), false);
+
+        // if isRenouncedToken flag is set burn owner role, else transfer ownership to specified admin
+        if (PIMConfig.isRenouncedIssuanceToken) {
+            _transferTokenOwnership(issuanceToken, address(0));
+        } else {
+            _transferTokenOwnership(issuanceToken, PIMConfig.admin);
+        }
+
+        // if isRenouncedWorkflow flag is set factory keeps admin rights over workflow, else transfer admin rights to specified admin
+        if (PIMConfig.isRenouncedWorkflow) {
+            // record the admin as fee recipient eligible to claim buy/sell fees
+            _pimFeeRecipients[fundingManager] = PIMConfig.admin;
+        } else {
+            _transferWorkflowAdminRights(orchestrator, PIMConfig.admin);
+        }
+
+        emit IPIM_WorkflowFactory_v1.PIMWorkflowCreated(
+            fundingManager,
+            address(issuanceToken),
+            _msgSender(),
+            PIMConfig.recipient,
+            PIMConfig.isRenouncedIssuanceToken,
+            PIMConfig.isRenouncedWorkflow
+        );
+
+        return (orchestrator, issuanceToken);
+    }
+
+    //--------------------------------------------------------------------------
+    // Permissioned Functions
+
+    /// @inheritdoc IPIM_WorkflowFactory_v1
+    function withdrawPimFee(address fundingManager, address to)
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        uint amount =
+            IBondingCurveBase_v1(fundingManager).projectCollateralFeeCollected();
+        IBondingCurveBase_v1(fundingManager).withdrawProjectCollateralFee(
+            to, amount
+        );
+        emit IPIM_WorkflowFactory_v1.PimFeeClaimed(_msgSender(), amount);
+    }
+
+    /// @inheritdoc IPIM_WorkflowFactory_v1
+    function transferPimFeeEligibility(address fundingManager, address to)
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        _pimFeeRecipients[fundingManager] = to;
+        emit IPIM_WorkflowFactory_v1.PimFeeRecipientUpdated(_msgSender(), to);
```

Comment:
Is it not important to know, for which `fundingManager` the `pimFeeRecipient` was updated?